src/CreateList.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import CreateList from './CreateList';
+
+test('Creates li for a task with the correct value', () => {
+    let fakeJson = new Array<any>()
+    let foo: any = {'task': 'first task'}
+    fakeJson.push(foo)
+    render(<CreateList list={fakeJson}/>)
```

Comment:
Can you put in a little whitespace for the test here. Think "Arrange, Act, Assert" - so it's 5-8 is Arranging or setting up the test, 9 you render (Act), and 10-11 is the assert.

Reply:
Sounds good